README.md

```diff
@@ -2,369 +2,143 @@
 
-o-header-services supports related content (eg Sign in or licence numbers). At large screen widths these appear to the far right of the header.
-At smaller screen widths this will be included in the drop down navigation.
+_Note: If there is a 'Sign in' option, it should be consistent with other FT products by aligning itself to the right and avoiding different wording such as 'Log in' or 'Login'._
 
-#### Core experience of the drawer
+You can see an example of a title section-only header and the markup for it in the [Origami Registry](https://registry.origami.ft.com/components/o-header-services#demo-title-section).
 
-Small screen users should still be able to access the contents of the drop down navigation even if their browser doesn't cut the mustard or the JavaScript has failed to load. This is not part of `o-header-services`, but we recommend you have the contents of the drop down navigation at the bottom of the page in a footer that is only visible if the body has a `.core` class. In core experience the hamburger menu should link to an anchor at the bottom of the page.
+### Primary Navigation
 
-To add support for related content, add the following to your markup:
+The primary navigation is an **optional** addition to the title section. It's most useful for high-level navigation.
 
-```diff
-<header class='o-header-services' data-o-component='o-header'>
-	<div class='o-header-services__top'>
-+		<div class="o-header-services__hamburger">
-+			<a class="o-header-services__hamburger-icon" href={{href}} role="button">
-+				<span class="o-header-services__visually-hidden">Open primary navigation</span>
-+			</a>
-+		</div>
-		<div class='o-header-services__logo'></div>
-		<div class='o-header-services__title'>
-			<h1 class='o-header-services__product-name'><a href=''>Tool or Service name</a></h1><span class='o-header-subrand__product-tagline '>Tagline to explain the product here</span>
-		</div>
-+		<ul class="o-header-services__related-content">
-+			<li>
-+				<a href={{href}}>XXXX</a>
-+			</li>
-+			<li >
-+				<a href={{href}}>Sign in</a>
-+			</li>
-+		</ul>
-	</div>
-</header>
-```
+This section of the header has specific behaviour, as it turns into a drawer at smaller viewport sizes (740px down). It requires the addition an extra element in the title section to support the hamburger button.
 
-### Primary navigation
+If you are using extra content (such as a 'Sign in' link), that will be pulled into the drawer, as well.
 
-If your application has more than one page you may want to add the primary navigation bar.
-This requires the drawer code, as seen above, and the following addition:
+You can see an example of the primary navigation and its markup in the [Origami Registry](https://registry.origami.ft.com/components/o-header-services#demo-primary-navigation).
```

Comment:
👇 Totally trivial, but putting "primary navigation" within the link makes the README more easy to scan.
```suggestion
You can see an example, with markup, of the [primary navigation in the Origami Registry](https://registry.origami.ft.com/components/o-header-services#demo-primary-navigation).
```